### DISCUSSION TITLE
Denote 23.03 fac_Country's countryCode as countryCode2

### DISCUSSION THREAD

**Discussion by pawsey-kbuckley** [opendcim/openDCIM#1399] [Ideas]
Just saw that part of the 23.03 (GPDR) work introduces a Country table into the DB.

```
+CREATE TABLE fac_Country (
+  countryCode CHAR(2) NOT NULL,
+  countryName VARCHAR(80) NOT NULL,
+  PRIMARY KEY (countryCode)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Was thinking that, given that you are scraping ISO 3116 data, you might denote the 
fact that you are using the ISO-3166 alpha2 data there, by using `countryCode2`, 
just in case (tm) you ever come to store the 3-char codes in the table as well.

Alternatively, if you are using the FIPS code, as opposed to ISO-3166 alpha2, then
maybe denote that provenance in some way, say `countryCodeFips` ?

Just a bit of "future-proofing" I guess.

**Comment by pawsey-kbuckley** [comment]
Thought I'd try and link to the one to the issue to which it pertains: #1382

**Comment by samilliken** [comment]
I don't really see any value in this as it is nothing other than a lookup table for a select list within the UI.   Other than pedantry, there is no value served.
